css/properties/text-anchor.json

```
@@ -10,12 +10,14 @@
           ],
           "support": {
             "chrome": {
-              "version_added": "≤80"
+              "version_added": "≤15"
```

```suggestion
              "version_added": "1"
```

See: https://github.com/caugner/css-properties/blame/7ff304e2b7a29845b2ef90a61e07a3629a5362b8/properties.css#L336